src/System.Security.Permissions/src/System.Security.Permissions.csproj

```diff
@@ -119,6 +119,12 @@
     <Reference Include="System" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="System.Collections" />
+    <Reference Include="System.Data.Common" />
```

Comment:
Hmm I added Data.Common but the others must have been added as I was trying other things. Diffs between my master and the upstream seem to be wrong. See note below about my master.